docs/releases/1.10.1.txt

```diff
@@ -56,3 +56,5 @@ Bugfixes
 * Reallowed the ``{% for %}`` tag to unpack any iterable (:ticket:`27058`).
 
 * Fixed ``makemigrations`` crash if a database is read-only (:ticket:`27054`).
+
+* Fixed duplicated managers in `Model._meta.managers` (:ticket:`27073`).
```

Comment:
Actually I'm gonna change "Fixed" to "Removed", it better reflects that it wasn't a bug to start with (although changing it fixes a bug in the check framework without affecting negatively the rest of the project).